README.md

```diff
@@ -39,7 +39,7 @@ Presto comes with sample configuration that should work out-of-the-box for devel
 
 * Main Class: `com.facebook.presto.server.PrestoServer`
 * VM Options: `-ea -XX:+UseG1GC -XX:G1HeapRegionSize=32M -XX:+UseGCOverheadLimit -XX:+ExplicitGCInvokesConcurrent -Xmx2G -Dconfig=etc/config.properties -Dlog.levels-file=etc/log.properties`
-* Working directory: `$MODULE_DIR$`
+* Working directory: `$MODULE_WORKING_DIR$`
```

Comment:
@xumingming Which version of IntelliJ are you using?

Reply:
Not the newest one, I checked on the web, seems MODULE_DIR is indeed deprecated, check it out here: https://blog.jetbrains.com/idea/2018/05/intellij-idea-2018-1-4-is-released/

> To avoid further misunderstanding, the $MODULE_DIR$ macro is deprecated.

> We’ve introduced 2 new macros:
> $MODULE_WORKING_DIR$ for module content directory.
> $MODULE_IML_DIR$ for directory containing the .iml file.

Reply:
@xumingming Maybe they might have re-introduced it in a later release, I am using 2020.x and $MODULE_DIR$ works fine. Also, it is still mentioned in the documentation (https://www.jetbrains.com/help/idea/absolute-path-variables.html). 

Maybe you can update the readme to say we can use either of these two. 
Working directory: `$MODULE_DIR$` or `$MODULE_WORKING_DIR$`

Reply:
Sounds good enough for me. Updated the patch